tests/plugins/autolink/manual/link.md

```diff
@@ -0,0 +1,12 @@
+@bender-ui: collapsed
```

Comment:
Please update this manual test styling according to [our style guide](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#31-manual-tests).

Reply:
Adjust style to newest guides: https://github.com/ckeditor/ckeditor4/pull/2518/commits/94363bad1d3ce4ed51b6c261bc8988cb4d265347

Also, I reformat all manual tests: https://github.com/ckeditor/ckeditor4/pull/2518/commits/a57e0137534c60559ce5e85da8c5b76fd84127eb and https://github.com/ckeditor/ckeditor4/pull/2518/commits/ef3e02ddb8a46a3ed3b4c1889d4693ed24fdceb4